bazel/swift_header_collector.bzl

```diff
@@ -0,0 +1,23 @@
+"""
+Propagate the generated Swift header from a swift_library target
+
+This exists to work around https://github.com/bazelbuild/rules_swift/issues/291
+"""
+
+def _swift_header_collector(ctx):
+    dep = ctx.attr.deps[0]
+    return [
+        DefaultInfo(
+            files = dep[CcInfo].compilation_context.headers,
+        ),
+    ]
+
+swift_header_collector = rule(
+    attrs = dict(
+        deps = attr.label_list(
```

Comment:
it looks like it could be a single `label`, does it need to be a `label_list`? Maybe if it was named `library` or `root` or something?

Reply:
Yea I was following the general convention but I agree it's weird, I'll replace with `library`